app/src/main/java/at/team30/setroute/ui/route_detail/RouteDetailFragment.kt

```diff
@@ -31,11 +33,25 @@ class RouteDetailFragment : Fragment() {
         super.onViewCreated(view, savedInstanceState)
 
         val route = viewModel.getRoute(args.routeId)
+
+        val imageView = view.findViewById<ImageView>(R.id.static_map_view);
+        val headerView = view.findViewById<ImageView>(R.id.header)
```

Comment:
We should probably remove the old `headerView` and replace it with some loading indicator.